ImageFeed/Helpers/UIColor+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UIColor+Extensions.swift
+//  ImageFeed
+//
+//  Created by Valeriy on 09.05.2023.
+//
+
+import UIKit
+
+extension UIColor {
+    static var ypBlack: UIColor { UIColor(named: "YP Black") ?? UIColor.black }
```

Comment:
swift по контексту умеет выводить типы, если в первом выражении указан тип, то при обращении к статическим свойствам его упоминание можно упростить
```suggestion
    static var ypBlack: UIColor { UIColor(named: "YP Black") ?? .black }
```

Еще обращу внимание на будущее, не стоит использовать названия файлов (все ассеты/ресурсы это файлы) с пробелами. Так не принято в разработке не просто так.